packages/core/directives/map.ts

```diff
@@ -363,7 +379,8 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
       fullscreenControlOptions: this.fullscreenControlOptions,
       mapTypeId: this.mapTypeId,
       clickableIcons: this.clickableIcons,
-      gestureHandling: this.gestureHandling
+      gestureHandling: this.gestureHandling,
+      tilt: this.tilt
```

Comment:
can you add a trailing comma
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas